src/styles/blocks/card.css

```diff
@@ -51,11 +50,9 @@
 .card__review-text {
   display: flex;
 
-  font-family: Roboto, sans-serif;
   font-weight: 400;
   font-size: 10px;
   line-height: 16px;
-
   color: #000;
```

Comment:
@IoghiStudio the comment about the colour was due to it's actually NOT black
<img width="659" alt="color" src="https://user-images.githubusercontent.com/39308717/200775414-3622c544-f40b-4b54-bfe2-9be5cffb84d0.png">

and moreover - what is the default font colour?
Yep, it's black, so i hardly imagine why would you state the default style.)